test/index.js

```diff
@@ -173,7 +177,7 @@ describe('.serve', () => {
 
         it('should serve files without a hash tag', done => {
             http.get('http://localhost:12321/index.js', res => {
-                res.headers['cache-control'].includes('max-age=0').should.be.true();
+                res.headers['cache-control'].includes('max-age=7200').should.be.true();
```

Comment:
I'd still like to have a test for `max-age=0`, which is the default.

Reply:
The tests on lines 97 and 136 already check `max-age=0`
This test suite is meant to check custom `max-age` for non-hashed assets